share/oneprovision/examples/example_packet.yaml

```diff
@@ -0,0 +1,70 @@
+---
+# Ansible playbook to configure hosts
+playbook: "default"
+
+# Cluster to group all the resources
+cluster:
+  name: "PacketCluster"
+
+# Defaults sections with information related with Packet
+defaults:
+#  provision:
+#    driver: "packet"
+#    packet_token: "**************************"
+#    packet_project: "************************"
+#    facility: "ams1"
+#    plan: "baremetal_0"
+#    os: "centos_7"
```

Comment:
- Have provision section uncommented
- We need also disable NAT (otherwise it breaks alias_sdnat for public networking)

```
  configuration:
    iptables_masquerade_enabled: False
```